main.go

```diff
@@ -86,19 +84,19 @@ func parse(line string) {
 	case strings.HasPrefix(trimmed, "?"):
 		c = nil
 
-	// success
+	// succeeded
```

Comment:
This change isn't necessary.

Reply:
Well just trying to keep the inline documentation as uniform as the code by using the same grammar

Reply:
Proposed change removed